src/frontend/lib/exif.ts

```diff
@@ -8,6 +8,8 @@ export const PhotoEXIFSchema = v.object({
   DateTime: v.optional(v.string()),
   ExposureTime: v.optional(v.number()),
   Flash: v.optional(v.number()),
+  // @ts-expect-error Need changes to schema to fix
+  FNumber: v.optional(v.number()),
```

Comment:
this is available in latest schema (1.7.0) but core doesn't support the schema just yet.